### PR TITLE
chore(test): add camel core cdi test

### PR DESCRIPTION
**Pull request by lburgazzoli**

### CODE REVIEW

File: integration-tests/core-cdi/pom.xml [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+        </dependency>
+
+        <!-- test dependencies -->
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-junit5</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.glassfish</groupId>
+            <artifactId>javax.json</artifactId>
```
Comment (ppalaga) [inline integration-tests/core-cdi/pom.xml]: Is javax.json really required?

File: integration-tests/core-cdi/pom.xml [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+            <artifactId>rest-assured</artifactId>
+        </dependency>
+
+    </dependencies>
+
+    <build>
+        <resources>
+            <resource>
+                <directory>src/main/resources</directory>
+                <filtering>true</filtering>
```
Comment (ppalaga) [inline integration-tests/core-cdi/pom.xml]: Is the resource filtering really required? I know we have it elsewhere and I always wondered whether it is useful.

File: integration-tests/core-cdi/pom.xml [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+            </resource>
+        </resources>
+        <plugins>
+            <!-- Skip enforcer plugin as we want to use jboss marshalling for test class -->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <configuration>
+                    <skip>true</skip>
```
Comment (ppalaga) [inline integration-tests/core-cdi/pom.xml]: Nitpick: An `<enforcer.skip>true</enforcer.skip>` property would be less code.

**Comment by lburgazzoli** [comment]
done

**Pull request merged**